flow/Makefile

```diff
@@ -129,7 +129,10 @@ SHELL          := /usr/bin/env bash
 # - the following settings allowed user to point OpenROAD binaries to different
 #   location
 # - default is current install / clone directory
-export FLOW_HOME ?= $(shell pwd)
+ifeq ($(origin FLOW_HOME), undefined)
```

Comment:
Nice find! How about a make macro? This problem is endemic as you found....

Reply:
I'm not sure a macro improves the readability and there are some very complex interactions around `$(`, `:=` and macro evaluation?

Reply:
Arcane syntax is a job for ChatGPT....